content/pom.xml

```diff
@@ -193,6 +198,67 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>cloud</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
```

Comment:
Instead of copying I would rather use ‘prefix’ (http://jackrabbit.apache.org/filevault-package-maven-plugin/generate-metadata-mojo.html#prefix) together with adjusting http://jackrabbit.apache.org/filevault-package-maven-plugin/package-mojo.html#jcrRootSourceDirectory